package.json

```diff
@@ -332,7 +332,7 @@
     "@metamask/post-message-stream": "^8.0.0",
     "@metamask/ppom-validator": "0.35.1",
     "@metamask/preinstalled-example-snap": "^0.2.0",
-    "@metamask/profile-sync-controller": "^2.0.0",
+    "@metamask/profile-sync-controller": "patch:@metamask/profile-sync-controller@npm%3A2.0.0#~/.yarn/patches/@metamask-profile-sync-controller-npm-2.0.0-02e028ab48.patch",
```

Comment:
Patched until [profile-sync-controller changes](https://github.com/MetaMask/core/pull/5013) are released.

The next upgrade of this package has an interdependency with the network controller.